lib/log_decorator/logging.rb

```diff
@@ -0,0 +1,33 @@
+module LogDecorator
+  module Logging
+    module ClassMethods
+      def instance_logger
+        @instance_logger ||= LogProxy.new(name, '#')
+      end
+
+      def _log
+        @_log ||= LogProxy.new(name, '.')
+      end
+
+      def _logger=(logger)
+        _log.logger = logger
+        instance_logger.prefix = logger
```

Comment:
Is this right?  This seems like a bug...I would expect `instance_logger.logger = logger`